_posts/2024-03-06-bitcoin-inquisition-#39.md

```diff
@@ -6,34 +6,57 @@ link: https://github.com/bitcoin-inquisition/bitcoin/pull/39
 permalink: /bitcoin-inquisition-39
 authors: [0xBEEFCAF3, EthanHeilman]
 components: ["consensus"]
-host: EthanHeilman
+host: EthanHeilman, arminsdev aka 0xBEEFCAF3
 status: upcoming
 commit:
 ---
 
 _Notes and questions to follow soon!_
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+The following pull request (PR) reinstates the usage of OP_CAT in accordance with the specifications outlined in the draft BIP, which can be found [here](https://github.com/bitcoin/bips/pull/1525). Reviewers are encouraged to familiarize themselves with the BIP before examining the code.
```

Comment:
Maybe offer some more background on the motivation for this PR? What can be built with it, etc.